github.sgnl.yaml

```diff
@@ -2,15 +2,21 @@
 description: "GitHub as a System of Record"
-# Enterprise Cloud: https://api.github.com/graphql 
-# Enterprise Server: {hostname}/api/graphql
+# Enterprise Cloud: api.github.com
+# Enterprise Server: Use your own hostname
 address: "{{Input Required}}"
 defaultSyncFrequency: HOURLY
 defaultSyncMinInterval: 1
 defaultApiCallFrequency: SECONDLY
 defaultApiCallMinInterval: 1
 type: "GitHub-1.0.0"
-adapterConfig: "ewogICAgImVudGVycHJpc2VTbHVnIjogInt7SW5wdXQgUmVxdWlyZWR9fSIKfQ==" # { "enterpriseSlug": "{{Input Required}}" }
+# Example Config:
+# {
+#    "enterpriseSlug": "{{Input Required}}",
+#    "isEntepriseCloud": "{{Input Required}}",
+#    "apiVersion": "{{Input Required for REST-based entities}}"
```

Comment:
nit: We give this example comment so that the user has some idea about datatype.

```suggestion
#    "enterpriseSlug": "your-enterprise-slug",
#    "isEntepriseCloud": true / false,
#    "apiVersion": "v1"
```

Reply:
fixed